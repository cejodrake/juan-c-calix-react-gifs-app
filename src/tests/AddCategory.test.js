import React from 'react'
import {shallow } from  'enzyme'
import {AddCategory} from '../components/AddCategory'


describe('test in componente AddCategory', ()=> {
const setCategories = ()=>{

}


  test(' debe de mostrarse correctament ', ()=>{
    const wrapper = shallow(<AddCategory/>)
    expect(wrapper).toMatchSnapshot();
  })

})
