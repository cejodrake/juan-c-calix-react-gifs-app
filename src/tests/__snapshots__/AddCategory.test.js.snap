// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in componente AddCategory  debe de mostrarse correctament  1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
